### ISSUE TITLE
What's the difference between the result of evaluation on Davis17 in the paper and ModelZoo?

### ISSUE THREAD

**Issue by MUVGuan** [yoxu515/aot-benchmark#22]
Hello! I find the J&F mean of AOTT(Y) on Davis17 validation set in Table1(b) in the paper is 78.2, while the J&F mean of AOTT in ModelZoo is 79.2, could you please tell me the difference?

**Comment by z-x-yang** [comment]
In this repository, AOT is re-implemented by using PyTorch, and the performance is slightly higher than the paper.

Not sure why this happens. There should be no difference in network architectures.

**Comment by MUVGuan** [comment]
Thanks for your reply. After traing, there are two ckpts were obtained, one is ckpt and another is ema_ckpt, which one do you use to report the evaluation result in the paper?

**Comment by MUVGuan** [comment]
Thanks for your reply. After traing, there are two ckpts were obtained, one is ckpt and another is ema_ckpt, which one do you use to report the evaluation result in the paper?

**Comment by z-x-yang** [comment]
ema_ckpt.
